src/items/LinkItem.js

```diff
@@ -47,14 +77,42 @@ const LinkItem = ({
   const url = extra?.url;
   const name = item.get('name');
 
+  const handleLoad = () => {
+    setIsLoading(false);
+  };
+
   const component = (
-    <iframe
-      id={id}
-      className={classes.iframe}
-      title={name}
-      src={url}
-      height={height}
-    />
+    <React.Fragment>
+      <div
+        hidden={!isLoading}
+        className={classes.iframeContainer}
+        style={{ height: height || '100%' }}
+      >
+        Link is Loading.
```

Comment:
Provide a prop to define a custom message. In our case it will allow us to provide some translation.